### PR TITLE
Updating sap_system to use naming module (anydb+ app tier)

### PR DESCRIPTION
**Pull request by KimForss**
## Problem
AnyDB updated to use naming module

## Solution
<Please elaborate the solution for the problem>

## Tests
<Please provide steps to test the PR>

## Notes
This branch is based on the hana + app tier branch so it should only contain the deltas needed for anydb

### CODE REVIEW

File: deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf [hunk 1]
```diff
@@ -14,64 +14,28 @@ variable "ppg" {
...
 
   # SAP vnet
   var_infra       = try(var.infrastructure, {})
   var_vnet_sap    = try(local.var_infra.vnets.sap, {})
-  vnet_sap_exists = try(local.var_vnet_sap.is_existing, false)
-  vnet_sap_arm_id = local.vnet_sap_exists ? try(local.var_vnet_sap.arm_id, "") : ""
-  vnet_sap_name   = local.vnet_sap_exists ? try(split("/", local.vnet_sap_arm_id)[8], "") : try(local.var_vnet_sap.name, "sap")
+  vnet_sap_arm_id = try(local.var_vnet_sap.arm_id, "")
+  vnet_sap_exists = length(local.vnet_sap_arm_id) > 0 ? true : false
+  vnet_sap_name   = local.vnet_sap_exists ? try(split("/", local.vnet_sap_arm_id)[8], "") : try(local.var_vnet_sap.name, "")
```
Comment (rukawata) [inline deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf:38]: If you decide to remove default for sap vnet name, please:
1. add this field to the default template `sapsystem.json`.
2. add this field with value in pipeline to make sure the test passes.

File: deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf [hunk 2]
```diff
@@ -14,64 +14,28 @@ variable "ppg" {
...
-  region         = try(var.infrastructure.region, "")
-  environment    = lower(try(var.infrastructure.environment, ""))
-  sid            = upper(try(var.application.sid, local.anydb_sid))
-  codename       = lower(try(var.infrastructure.codename, ""))
-  location_short = lower(try(var.region_mapping[local.region], "unkn"))
-  // Using replace "--" with "-" and "_-" with "-" in case of one of the components like codename is empty
-  prefix    = try(local.var_infra.resource_group.name, upper(replace(replace(format("%s-%s-%s_%s-%s", local.environment, local.location_short, substr(local.vnet_sap_name_prefix, 0, 7), local.codename, local.sid), "_-", "-"), "--", "-")))
-  sa_prefix = lower(replace(format("%s%s%sdiag", substr(local.environment, 0, 5), local.location_short, substr(local.codename, 0, 7)), "--", "-"))
+  region  = try(var.infrastructure.region, "")
+  sap_sid = upper(try(var.application.sid, ""))
```
Comment (rukawata) [inline deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf:29]: There is a logic change from 
`sid            = upper(try(var.application.sid, local.anydb_sid))`
to this.
This means, if just deploy an Oracle database, there is no `sid`, no `sap_sid` - will this not cause any problem?
Maybe try to deploy without application and just an Oracle database and see what happens.

**Review by rukawata — COMMENTED** [review]
Most part looks good! See below comments:

File: deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf (hunk 1 above)
Comment (KimForss) [reply deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf:38]: Done

File: deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf (hunk 2 above)
Comment (KimForss) [reply deploy/terraform/terraform-units/modules/sap_system/anydb_node/variables_local.tf:29]: There should not be any problems without sap_sid - will validate though

**Review by rukawata — APPROVED** [review]
lgtm!

**Pull request merged**